src/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const authEmulatorHost = 'http://localhost:9099/'
```

Comment:
constants should be written in uppercase

Reply:
I declared a new variable to simplify the naming: `EMULATOR.hosts.auth` (or `.db` or `.functions`)